### PR TITLE
DLS-7029: Change Activity CYA page

### PR DESCRIPTION
**Pull request by drewmiley**

### CODE REVIEW

**Comment by drewmiley** [comment]
Initial roughly working Change Activity CYA page

- Tests need to be improved
- Return Period
- Third Party Packaging need to be added

**Comment by drewmiley** [comment]
withouttppcya

File: app/views/changeActivity/ChangeActivityCYAView.scala.html [hunk 1]
```diff
@@ -16,18 +16,29 @@
...
+    }
 
-   @for((heading, summaryList) <- seqSummaryList) {
-     <h2 class="govuk-heading-m">@heading</h2>
-   @govukSummaryList(summaryList)
-}
+    @formHelper(action = submitCall, Symbol("autoComplete") -> "off") {
+        @govukButton(ButtonViewModel(messages("updateRegisteredDetails.checkYourAnswers.submit")))
+    }
 }
```
Comment (ginnyamazed) [inline app/views/changeActivity/ChangeActivityCYAView.scala.html:44]: See screenshot - for CYA - amount-produced = None, old service shows Use third party packagers but does not show Packaged in the UK.  
Screenshot 2023-07-11 at 15 12 29

**Review by ginnyamazed — CHANGES_REQUESTED** [review]

File: app/views/changeActivity/ChangeActivityCYAView.scala.html (hunk 1 above)
Comment (drewmiley) [reply app/views/changeActivity/ChangeActivityCYAView.scala.html:44]: I will investigate/fix this. The answers on those last three sections are possibly defaulting to No - so I'd need to solve that in addition if so.

File: app/views/changeActivity/ChangeActivityCYAView.scala.html [hunk 2]
```diff
@@ -16,18 +16,29 @@
...
+    seqSummaryList: Seq[(String, SummaryList)],
+    submitCall: Call
+)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = titleNoForm(messages("checkYourAnswers.title"))) {
+@layout(pageTitle = titleNoForm(messages("changeActivity.checkYourAnswers.title"))) {
 
-    <h1 class="govuk-heading-l">@messages("checkYourAnswers.heading")</h1>
+    <span class="govuk-caption-l" id="pre-header-caption"> @alias - @returnPeriod </span>
+    <h1 class="govuk-heading-l">@messages("changeActivity.checkYourAnswers.heading")</h1>
```
[the remark this answers is not in the document]
Comment (drewmiley) [reply app/views/changeActivity/ChangeActivityCYAView.scala.html]: Discussed and changed

File: app/views/changeActivity/ChangeActivityCYAView.scala.html (hunk 1 above)
Comment (drewmiley) [reply app/views/changeActivity/ChangeActivityCYAView.scala.html:44]: Fixed this

**Review by ginnyamazed — APPROVED** [review]

**Pull request merged**